deploy/helm/templates/l7-envoyfilter.yaml

```diff
@@ -18,6 +18,12 @@ spec:
             protocol: TCP
             address: 0.0.0.0
             port_value: 15019
+        additional_addresses:
+        - address:
+            socket_address:
+              protocol: TCP
+              address: "::"
```

Comment:
will it fail to start in ipv4 env

Reply:
k8s enable dual stack by default starting from 1.21, ref: https://kubernetes.io/docs/concepts/services-networking/dual-stack/